.gitignore

```diff
@@ -0,0 +1,6 @@
+test.md
+.hintrc
+node_modules/
+package-lock.json
+package.json
+.stylelintrc.json
```

Comment:
- [ ] [OPTIONAL] Kindly remove all files and leave test.md & node_modules in the gitignore